suite/suite.go

```diff
@@ -193,12 +232,13 @@ func runTests(t testing.TB, tests []testing.InternalTest) {
 
 	r, ok := t.(runner)
 	if !ok { // backwards compatibility with Go 1.6 and below
```

Comment:
We don't officially support Go 1.6 anymore. Can we remove this check altogether or at least remove the old backwards compatibility change and update this to reflect 1.18 and fuzzing?

Reply:
Sounds like a good idea. Should I create a separate change to remove this and rebase on top it?